[CI] IndexingIT testTsdb {upgradedNodes=3} failing

**Build Scans:**
- [elasticsearch-intake #13138 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/edezft6t4frle)
- [elasticsearch-intake #13139 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/o473lltfahdpe)
- [elasticsearch-pull-request #41713 / 9.0.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/uhjq723zit2aa)
- [elasticsearch-pull-request #41640 / 8.17.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/c52j2znt6psai)
- [elasticsearch-pull-request #41624 / 8.17.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/psw6zqiyjjv5s)

**Reproduction Line:**
```
./gradlew ":qa:rolling-upgrade:v9.0.0#bwcTest" -Dtests.class="org.elasticsearch.upgrades.IndexingIT" -Dtests.method="testTsdb {upgradedNodes=2}" -Dtests.seed=124BB3C25C670031 -Dtests.bwc=true -Dtests.locale=ko-KR -Dtests.timezone=Asia/Muscat -Druntime.java=23
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.upgrades.IndexingIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testTsdb%20%7BupgradedNodes%3D3%7D),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Expected a map containing
           aggregations: a map containing
                    tsids: a map containing
                    buckets: a list containing
                            0: a map containing
                            key: expected "JFFUy14C9UcX3MnFnsFrpf0UyQYeoe87sMXUQ025sHhvhKDa4g" but was "JFFUy14C9UcX3MnFnsFrpf1fK0gnGg01X2P0BuJX0wb-7i2pqA"
                            avg: a map containing
                            value: expected a numeric value within <0.5> of <218.95> but <21895.0> differed by <21675.55> more than delta <0.5>
                      doc_count: <4380> unexpected but ok
                            1: a map containing
                            key: expected "JFFUy14C9UcX3MnFnsFrpf1fK0gnGg01X2P0BuJX0wb-7i2pqA" but was "JFFUy14C9UcX3MnFnsFrpf0UyQYeoe87sMXUQ025sHhvhKDa4g"
                            avg: a map containing
                            value: expected a numeric value within <0.005> of <21895.0> but <217.47068965517755> differed by <21677.524310344823> more than delta <0.005>
                      doc_count: <4350> unexpected but ok
                            2: a map containing
                            key: expected "JFFUy14C9UcX3MnFnsFrpf1qe4IAVEfi3wL8wnx3pd_rA41HpA" but was "JFFUy14C9UcX3MnFnsFrpf3X1Mw5gSg0zb_Y50PDlARq0q0ANA"
                            avg: a map containing
                            value: expected a numeric value within <0.005> of <2408.45> but <-217.47068965517755> differed by <2625.915689655177> more than delta <0.005>
                      doc_count: <4350> unexpected but ok
                            3: a map containing
                            key: expected "JFFUy14C9UcX3MnFnsFrpf3X1Mw5gSg0zb_Y50PDlARq0q0ANA" but was "JFFUy14C9UcX3MnFnsFrpf1qe4IAVEfi3wL8wnx3pd_rA41HpA"
                            avg: a map containing
                            value: expected a numeric value within <0.005> of <-218.95> but <4800.95> differed by <5019.8949999999995> more than delta <0.005>
                      doc_count: <30> unexpected but ok
doc_count_error_upper_bound: <0> unexpected but ok
        sum_other_doc_count: <0> unexpected but ok
                _shards: <{total=1, failed=0, successful=1, skipped=0}> unexpected but ok
                   hits: <{hits=[], total={value=10000, relation=gte}, max_score=null}> unexpected but ok
                   took: <94> unexpected but ok
              timed_out: <false> unexpected but ok
```

**Issue Reasons:**
- [main] 5 failures in test testTsdb {upgradedNodes=3} (0.7% fail rate in 687 executions)
- [main] 3 failures in step 9.0.0_bwc-snapshots (0.9% fail rate in 324 executions)
- [main] 2 failures in step 8.17.0_bwc-snapshots (0.7% fail rate in 277 executions)
- [main] 2 failures in pipeline elasticsearch-intake (2.4% fail rate in 84 executions)
- [main] 3 failures in pipeline elasticsearch-pull-request (1.1% fail rate in 269 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -254,41 +254,7 @@ tests:
 - class: org.elasticsearch.xpack.apmdata.APMYamlTestSuiteIT
   method: test {yaml=/10_apm/Test template reinstallation}
   issue: https://github.com/elastic/elasticsearch/issues/116445
-- class: org.elasticsearch.upgrades.DownsampleIT
-  method: testRollupIndex {upgradedNodes=3}
-  issue: https://github.com/elastic/elasticsearch/issues/117122
-- class: org.elasticsearch.upgrades.DownsampleIT
-  method: testRollupIndex {upgradedNodes=1}
-  issue: https://github.com/elastic/elasticsearch/issues/117123
-- class: org.elasticsearch.upgrades.DownsampleIT
-  method: testRollupIndex {upgradedNodes=2}
-  issue: https://github.com/elastic/elasticsearch/issues/117124
-- class: org.elasticsearch.upgrades.IndexingIT
-  method: testAutoIdWithOpTypeCreate {upgradedNodes=1}
-  issue: https://github.com/elastic/elasticsearch/issues/117125
-- class: org.elasticsearch.upgrades.IndexingIT
-  method: testTsdb {upgradedNodes=1}
-  issue: https://github.com/elastic/elasticsearch/issues/117126
-- class: org.elasticsearch.upgrades.IndexingIT
-  method: testIndexing {upgradedNodes=1}
-  issue: https://github.com/elastic/elasticsearch/issues/117127
-- class: org.elasticsearch.upgrades.IndexingIT
-  method: testSyntheticSource {upgradedNodes=1}
-  issue: https://github.com/elastic/elasticsearch/issues/117128
-- class: org.elasticsearch.upgrades.IndexingIT
-  method: testIndexing {upgradedNodes=3}
-  issue: https://github.com/elastic/elasticsearch/issues/117135
-- class: org.elasticsearch.upgrades.IndexingIT
-  method: testTsdb {upgradedNodes=3}
-  issue: https://github.com/elastic/elasticsearch/issues/117136
-- class: org.elasticsearch.upgrades.IndexingIT
-  method: testIndexing {upgradedNodes=2}
-  issue: https://github.com/elastic/elasticsearch/issues/117137
-- class: org.elasticsearch.upgrades.IndexingIT
-  method: testTsdb {upgradedNodes=2}
-  issue: https://github.com/elastic/elasticsearch/issues/117138
-- class: org.elasticsearch.upgrades.IndexingIT
-  issue: https://github.com/elastic/elasticsearch/issues/117140
+
 
 # Examples:
 #

